devfile.yaml

```diff
@@ -43,24 +40,17 @@ commands:
       commandLine: mvn clean install
       group:
         kind: build
```

Comment:
should the `mvn clean install` also be in the eap container?

Reply:
i don't think we need one more command to build project